OneSchedule.Updater/OneSchedule.Updater.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <UserSecretsId>1ad6de21-aeff-40eb-9c85-69c1e342737d</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Why is this file specified as Remove, but on the 14th row it is specified as Include? Decide which option you need